christina-neth/READ.ME

```diff
@@ -0,0 +1,24 @@
+# Project Name: Kilovolt Blog
+
+**Author**: Neth Horn, Christina Gislason
+**Version**: 1.0.0 
+
+## Overview
+
+This application is a blog in which articles can be filtered by choosing an author or category. The articles are generated using a template and each article can be expanded or collapsed.
+
+## Getting Started
+
+A user must be able to use HTML, JS, and apply Handlebars and jQuery to build this app on their machine. 
+
+## Architecture
+
+jQuery

```

Comment:
make sure to preview your markdown files. These 2 lines actually appear on the same line. I would suggest making it a list and adding `- ` at the beginning of these 2 lines